### ISSUE TITLE
Possession in Hungarian and Finnish

### ISSUE THREAD

**Issue by ftyers** [UniversalDependencies/docs#230]
Hey all, there is a discrepancy in how Hungarian and Finnish treat the absence of nominal possession:

```
3       vezér   vezér   NOUN    _       Case=Nom|Number=Sing|Number[psed]=None|Number[psor]=None|Person[psor]=None      4       nmod:att        _       _
```

vs.

```
7       johtaja johtaja NOUN    N       Case=Nom|Number=Sing    9       nsubj   _       _
```

Is this an intended difference, or a result of automatic conversion ? What is the preferred way in of dealing with features like this which are not expressed in UD ?

**Comment by vinczev** [comment]
Hi Fran,

The preferred way is not to include those features, that is, the Finnish 
way. The Hungarian version was due to automatic conversion, which has 
been corrected in v1.2 of the data.

Best,
Veronika

**Comment by ftyers** [comment]
Excellent news, thanks!
